.changeset/red-candles-invite.md

```diff
@@ -0,0 +1,21 @@
+---
+'@astrojs/sitemap': minor
+---
+
+Add a `customSitemaps` option to include extra sitemaps in the `sitemap-index.xml` file generated by Astro. This is useful for multi-framework setups on the same domain, where for example Astro generates `example.com` and some other framework handles `example.com/blog` and you want to include `example.com/blog/sitemap.xml` to `sitemap-index.xml`.
```

Comment:
```suggestion
Adds a `customSitemaps` option to include extra sitemaps in the `sitemap-index.xml` file generated by Astro.

This is useful for multi-framework setups on the same domain as your Astro site (`example.com`), such as an external blog at `example.com/blog` whose sitemap is generated by another framework.

The following example shows configuring your Astro site to include externally-created sitemaps for a blog and help centre along with the generated sitemap entries in `sitemap-index.xml`:
```
Please feel free to adjust if the nuance is not quite right, but this is kind of the structure we'd typically use!

Reply:
Looks good overall, thanks! Minor comments:
- `...such as an external blog at`: I think it should be `externally generated` or something like that. Because effectively the blog is not external, it is under the same domain. It's just created with smth other than Astro.
- `centre`:  I think it's generally written as `help center`. Or is it the british writing style? Sorry, I'm not sure.

Reply:
(Noting that I updated my suggestion above in response to Gourav's comments! I think this should be OK now!)

Reply:
@gouravkhunger Would you please edit the text as you'd like to see it, then I can simply review and check if there are any language issues to worry about? I think that's the easiest way to proceed here! :smile: